src/pages/about.js

```diff
@@ -63,25 +59,25 @@ export default function() {
           ))}
         </div>
       </PageSection>
-      <PageSection title="Event organizers">
+      <PageSection title={t("about.event-organizers-text")}>
         <div className="flex flex-wrap">
           {ORGANIZERS.map((organizer, index) => (
             <Avatar key={index} {...organizer} />
           ))}
         </div>
       </PageSection>
-      <PageSection title="Founders">
+      <PageSection title={t("about.founders-text")}>
         <div className="flex flex-wrap">
           {FOUNDERS.map((founder, index) => (
             <Avatar key={index} {...founder} />
           ))}
         </div>
       </PageSection>
       <PageHighlight
-        header="We don't tolerate idiotic behavior"
```

Comment:
Eliminé el uso de `"idiot"` aquí.

Ver: https://developers.google.com/style/inclusive-documentation#ableist-language
https://en.wikipedia.org/wiki/List_of_disability-related_terms_with_negative_connotations

Reply:
Go for it, ese copy es viejo, lo tenemos en Eventbrite también 😬 .